python/ray/serve/config.py

```diff
@@ -186,3 +186,12 @@ def _validate(self):
                 raise TypeError(
                     "resources in ray_actor_options must be a dictionary.")
             self.resource_dict.update(custom_resources)
+
+
+class CondaEnv(BaseModel):
+    """Configuration options for a conda environment.
+
+    :param name: The name of the conda environment.
+    :type name: str
+    """
```

Comment:
nit: slight preference to put this in its own file `env.py` where we can add more env-related things in the future

Reply:
Sounds good, I was debating this as well